protocols/netty/src/main/java/org/apache/james/protocols/netty/AbstractAsyncServer.java

```diff
@@ -184,13 +190,15 @@ public synchronized void unbind() {
         bossGroup.ifPresent(boss -> futures.add(boss.shutdownGracefully()));
 
         if (workerGroup != null) {
-            futures.add(workerGroup.shutdownGracefully());
+            futures.add(workerGroup.shutdownGracefully(100, 3000, TimeUnit.MILLISECONDS));
```

Comment:
Being able to differenciate the timeout from the quiet period indeed is a good finding!

I agree having a short quiet period, and longer timeouts. Though 100ms looks low, maybe 500ms - personal tastes...

Also we need to name magic constants here so that the intent can be understood without having to pro-actively read the Netty documentation.

```
futures.add(workerGroup.shutdownGracefully(SHUTDOWN_QUIET_PERIOD, SHUTDOWN_TIMEOUT, 3000, TimeUnit.MILLISECONDS));
```